src/intl/en/page-what-is-ether.json

```diff
@@ -0,0 +1,96 @@
+  "page-what-is-ether-how-to-send-and-receive-eth-description-5": "Think of your <strong>account like your email address</strong> where people send you money.",
+  "page-what-is-ether-how-to-send-and-receive-eth-description-6": "Think of your <strong>wallet like your email app</strong> where you check balances and make payments.",
+  "page-what-is-ether-how-to-send-and-receive-eth-description-7": "For larger amounts, consider hardware wallets for added security.",
+  "page-what-is-ether-how-to-send-and-receive-eth-description-8": "Learn about Ethereum and how it works.",
+  "page-what-is-ether-how-long-does-it-take-to-send-eth": "How long does it take to send ETH?",
+  "page-what-is-ether-how-long-does-it-take-to-send-eth-description-1": "ETH transactions typically complete in under 30 seconds. The Ethereum network processes blocks every 12 seconds, though transactions may queue during congestion.",
+  "page-what-is-ether-how-long-does-it-take-to-send-eth-description-2": "Transactions achieve 'finality' after ~15 minutes, compared to Bitcoin's 60-minute average.",
+  "page-what-is-ether-how-long-does-it-take-to-send-eth-description-3": "During high network traffic, you can speed up transactions by paying higher transaction fees, placing you ahead in line. Fees are split between validators and a burning mechanism.",
+  "page-what-is-ether-how-much-does-it-cost-to-send-eth": "How much does it cost to send ETH?",
+  "page-what-is-ether-how-much-does-it-cost-to-send-eth-description-1": "Ethereum transactions require transaction <strong>fees paid in ETH</strong> and calculated in a unit called gas. Fees fluctuate based on network demand, with off-peak periods offering lower costs.",
+  "page-what-is-ether-l2s": "Enter L2s: Scaling Ethereum",
+  "page-what-is-ether-l2s-description-1": "As Ethereum's popularity grows, keeping transaction fees low becomes challenging. Layer 2 (L2) networks address this issue.",
+  "page-what-is-ether-l2s-description-2": "L2s like <optimism>Optimism</optimism> and <arbitrum>Arbitrum</arbitrum> offer <strong>5-10x cheaper fees</strong> while inheriting Ethereum's security. They process transactions off-chain and post summaries to Ethereum.",
+  "page-what-is-ether-l2s-description-3": "Think of them as express lanes that provide faster, cheaper transactions alongside Ethereum's main highway.",
+  "page-what-is-ether-l2s-description-4": "L2 <strong>transfers can cost as little as $0.04</strong>, bringing Ethereum to millions more users through integrations with companies like Robinhood, PayPal, and Shopify.",
+  "page-what-is-ether-what-is-the-eth-supply": "What is the ETH supply?",
+  "page-what-is-ether-what-is-the-eth-supply-description-1": "Unlike Bitcoin's fixed 21 million cap, ETH has dynamic supply mechanics:",
+  "page-what-is-ether-what-is-the-eth-supply-description-2": "New ETH is issued to reward network validators",
```

Comment:
Would be nice to adjust wording to convey that this is rate-limited and based on amount of ETH staked.

Perhaps:

> New ETH is issued to reward network validators at a limited rate calculated by the protocol

May be too much for here though

Reply:
sounds good to me